tests/integration/workflows/fixtures/oai_arxiv_record_with_plots.xml

```diff
@@ -24,7 +24,7 @@
             </author>
           </authors>
           <title>Spectra generated by a confined softcore Coulomb potential</title>
-          <categories>math-ph math.MP</categories>
+          <categories>math-ph math.MP astro-ph astro-ph.IM</categories>
```

Comment:
This is what was missing to make it work, right? As I remarked elsewhere, this is not ideal, because it's "cheating" by faking a record that doesn't exists, inducing false security in our system. In fact, my plan is to complete #1773 and later tackle this test suite, so that we start trusting it more.